Ginger/Ginger/Actions/ActionEditPages/ActAccessibilityTestingEditPage.xaml.cs

```diff
@@ -0,0 +1,191 @@
+    {
+        private ActAccessibilityTesting mAct;
+        PlatformInfoBase mPlatform;
+        string mExistingPOMAndElementGuidString = null;
+        public ActAccessibilityTestingEditPage(ActAccessibilityTesting act)
+        {
+            InitializeComponent();
+            mAct = act;
+            if (act.Platform == ePlatformType.NA)
+            {
+                act.Platform = GetActionPlatform();
+            }
+            mPlatform = PlatformInfoBase.GetPlatformImpl(act.Platform);
+            List<eLocateBy> LocateByList = mPlatform.GetPlatformUIElementLocatorsList();
+            xElementLocateByComboBox.BindControl(mAct, Act.Fields.LocateBy, LocateByList);
+            xLocateValueVE.Init(Context.GetAsContext(mAct.Context), mAct.GetOrCreateInputParam(Act.Fields.LocateValue));
+
+            List<ActAccessibilityTesting.eTags> supportedControlActions = GetStandardTags();
+            //bind controls
+            GingerCore.General.FillComboFromEnumObj(xStandardComboBox, mAct.Standard, supportedControlActions.Cast<object>().ToList());
+            GingerCore.GeneralLib.BindingHandler.ObjFieldBinding(xStandardComboBox, ComboBox.SelectedValueProperty, mAct, ActAccessibilityTesting.Fields.Standard);
+
+            ValueUC.Init(Context.GetAsContext(mAct.Context), mAct.GetOrCreateInputParam("Value"));
+            xLocateValueVE.BindControl(Context.GetAsContext(mAct.Context), mAct, Act.Fields.LocateValue);
+            xTargetRadioButton.Init(typeof(ActAccessibilityTesting.eTarget), xTargetRadioButtonPnl, mAct.GetOrCreateInputParam(ActAccessibilityTesting.Fields.Target, ActAccessibilityTesting.eTarget.Page.ToString()), TargetRadioButton_Clicked);
+            if ((act.GetInputParamValue(ActAccessibilityTesting.Fields.Target) == ActAccessibilityTesting.eTarget.Element.ToString()))
+            {
+                xValueLabel.Visibility = System.Windows.Visibility.Collapsed;
+                ValueUC.Visibility = System.Windows.Visibility.Collapsed;
+                xLocateByAndValuePanel.Visibility = System.Windows.Visibility.Visible;
+            }
+            else
+            {
+                xValueLabel.Visibility = System.Windows.Visibility.Visible;
+                ValueUC.Visibility = System.Windows.Visibility.Visible;
+                xLocateByAndValuePanel.Visibility = System.Windows.Visibility.Collapsed;
+                xValueLabel.Content = "URL:";
+            }
+            SetLocateValueFrame();
+        }
+
+        private ePlatformType GetActionPlatform()
+        {
+            ePlatformType platform;
+            if (mAct.Context != null && (Context.GetAsContext(mAct.Context)).BusinessFlow != null)
+            {
+                string targetapp = (Context.GetAsContext(mAct.Context)).BusinessFlow.CurrentActivity.TargetApplication;
+                platform = (from x in WorkSpace.Instance.Solution.ApplicationPlatforms where x.AppName == targetapp select x.Platform).FirstOrDefault();
```

Comment:
please use WorkSpace.Instance.Solution.ApplicationPlatforms.FirstOrDefault() format to get the first item

Reply:
Updated